modules/persistent-storage-snapshots-starting-controller-provisioner.adoc

```diff
@@ -46,6 +46,12 @@ rules:
     resources: ["volumesnapshotdatas"]
     verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
 ----
+. As the cluster admin, provide the `hostNetwork security context` constraint (SCC):
```

Comment:
* cluster admin -> cluster administrator
* The backtick is in the wrong spot, should be only around "hostNetwork"